packages/ra-data-simple-rest/src/index.ts

```diff
@@ -10,7 +10,78 @@ import {
     UPDATE_MANY,
     DELETE,
     DELETE_MANY,
-} from 'react-admin';
+} from 'ra-core';
+
+interface Pagenation {
+    page: number;
+    perPage: number;
+}
+
+interface Sort {
+    field: string;
+    order: 'ASC' | 'DESC';
+}
+
+interface GetListRequestParams {
+    pagination: Pagenation;
+    sort: Sort;
+    filter: {
+        [key: string]: any;
+    };
+}
+
+interface IdParams {
```

Comment:
use the Identifier type from ra-core instead

Reply:
I'm sorry, I'll check it.